Backend/DinoTracker.Core/CreatePaleontologistCommand.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace DinoTracker.Core
+{
+    public class CreatePaleontologistCommand
+    {
+        private IPaleontologistRepository paleontologistRepository;
+        private string name;
+        private string username;
+        private string password;
+
+        public CreatePaleontologistCommand(IPaleontologistRepository paleontologistRepository, string name, string username, string password)
+        {
+            this.paleontologistRepository = paleontologistRepository;
+            this.name = name;
+            this.username = username;
+            this.password = password;
+        }
+
+        public void Execute()
+        {
+            List<Paleontologist> paleontologists = this.paleontologistRepository.FindPaleontologist(this.username).ToList();
+
+            if (paleontologists.Count > 0)
+                throw new Exception("username already in use");
+            else
+                this.paleontologistRepository.CreatePaleontologist(this.name, this.username, this.password);

```

Comment:
This doesn't have to hold up this PR but going forward, always use curly braces even for one line if elses... it saves you the headache in the future of debugging if someone forgets to add them when they add a second line to the block.